packages/hardhat-ethers/test/hardhat-ethers-provider.ts

```diff
@@ -77,7 +77,7 @@ describe("hardhat ethers provider", function () {
 
     it("should work for impersonated accounts", async function () {
       const [s] = await this.env.ethers.getSigners();
-      const randomAddress = "0xf965cceab9374d9f961581b6e38942e45e1cfeae";
```

Comment:
@fvictorio this might involve a breaking change in Hardhat.

Addresses in Eth need to conform to [EIP-2](https://eips.ethereum.org/EIPS/eip-2). Currently, we don't check this until we're trying to sign a message, which means the error might come at an inopportune time. There's an issue to improve this UX: https://github.com/NomicFoundation/edr/issues/260

The random addresses in the tests were invalid Ethereum addresses, but other users might start having issues with this as well, if `ethereumjs` wasn't testing the validity of addresses.